test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -410,9 +398,7 @@
                          INNER JOIN report_statuses rs ON r.status_id=rs.id
                          ORDER BY c.certname"
                         (sutils/sql-hash-as-str "r.hash")
-                        (sutils/sql-uuid-as-str "r.transaction_uuid")
-                        cast-to-jsonb
-                        cast-to-jsonb))]
+                        (sutils/sql-uuid-as-str "r.transaction_uuid")))]
```

Comment:
We can probably just get rid of this function and just use `::text` directly in the SQL like you have for `jsonb`